docs/changelog-fragments/1.feature.rst

```diff
@@ -0,0 +1,2 @@
+Add cython extension for libssh client
+API's initial commit - by :user:`ganeshrn`
```

Comment:
I've modified the documentation example to use double dash as in `--` instead of `-` (it's turned into an M-dash when rendered).
```suggestion
API's initial commit -- by :user:`ganeshrn`
```